frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings/tests/simple-embedding.unit.spec.tsx

```diff
@@ -73,6 +73,37 @@ describe("EmbeddingSdkSettings (EE with Simple Embedding feature)", () => {
     });
   });
 
+  it("should show embed button and docs when simple embedding is available", async () => {
+    await setup({
+      isEmbeddingSdkEnabled: false,
+      isEmbeddingSimpleEnabled: true,
+      showSdkEmbedTerms: false,
+    });
+
+    const card = screen
+      .getAllByTestId("sdk-setting-card")
+      .find((card) => card.textContent?.includes("Embedded Analytics JS"));
+
+    expect(card).toHaveTextContent("New embed");
+    expect(card).toHaveTextContent("Documentation");
+  });
+
+  it("should not show embed button and docs when simple embedding is not available", async () => {
+    await setup({
+      isEmbeddingSdkEnabled: false,
+      isEmbeddingSimpleEnabled: false,
+      showSdkEmbedTerms: false,
+      tokenFeatures: { embedding_sdk: false, embedding_simple: false },
+    });
+
+    const card = screen
+      .getAllByTestId("sdk-setting-card")
+      .find((card) => card.textContent?.includes("Embedded Analytics JS"));
+
+    expect(card).not.toHaveTextContent("New embed");
+    expect(card).not.toHaveTextContent("Documentation");
```

Comment:
I'd also assert that you should see the upsell label

<img width="930" height="224" alt="Image" src="https://github.com/user-attachments/assets/43a5ce91-15df-4096-b224-5107fd77d119" />